internal/net/control/control.go

```diff
@@ -55,6 +57,7 @@ const (
 	IPRecoverDestinationAddr
 )
 
+// New return the socket controller.
```

Comment:
I think that "returns" is more correct representation grammatically.

Reply:
```suggestion
// New returns the socket controller.
```